vignettes/MIRAvignette.Rmd

```diff
@@ -15,12 +15,15 @@ Note: This vignette gives a broad overview of the MIRA package and some main fun
 ## MIRA Overview
 The Methylation-based Inference of Regulatory Activity package, MIRA for short, aggregates DNA methylation data from regions across the genome that correspond to a regulatory element, giving a summary signature for that element. The shape of this signature and the associated score can be used as a metric to compare regulatory activity in different samples and conditions. MIRA works with genome-scale, single-nucleotide-resolution methylation data such as is generated by reduced representation bisulfite sequencing and whole genome bisulfite sequencing but overcomes the sparsity of methylation that can be found in such data by aggregating across many regions. Since methylation will generally be lower at the location of transcription factor binding compared to surrounding regions (exceptions exist of course), MIRA can be used to infer differential transcription factor activity based on changes in the deepness of the dip in the MIRA signature, which implies increased or decreased binding.
 
-```{r,echo=FALSE}
+```{r, echo = FALSE}
 library(MIRA)
 data(exampleBins)
 plotMIRARegions(exampleBins)
-exScores=exampleBins[,.(score = scoreDip(methyl,binCount=11)),by=.(featureID,sampleName)]
-exScores[,sampleType := c("Condition1","Condition2")][] #normally would do join with annoDF
+exScores = exampleBins[, .(score = scoreDip(methyl, binCount = 11)), 
+                       by = .(featureID, sampleName)]
+#normally would do join with annoDF 
```

Comment:
1. What are these comments for? What's annoDF?
2. I think comments should have spaces after `#`